### PR TITLE
change excel header name

### PR DESCRIPTION
**Pull request by sravfeyn**
@proteusvacuum http://manage.dimagi.com/default.asp?226126#edit_0_226126 (Wait for build)

### CODE REVIEW

**Comment by sravfeyn** [comment]
Kicked build again, it looks unrelated

File: corehq/apps/users/bulkupload.py [hunk 1]
```diff
@@ -36,17 +36,30 @@ class UserUploadError(Exception):
...
 
 def check_headers(user_specs):
+    messages = []
     headers = set(user_specs.fieldnames)
 
+    # Backwards warnings
+    for (old_name, new_name) in old_headers.iteritems():
+        if old_name in headers:
+            messages.append(
+                _("'The column header '{old_name}' is illegel, please use '{new_name}' instead.").format(
```
Comment (proteusvacuum) [inline corehq/apps/users/bulkupload.py]: This seems a little harsh. Maybe use the word "deprecated" or "no longer in use" instead? Either way, it's spelt 'illegal'.

**Comment by proteusvacuum** [comment]
:+1: looks good. will wait on @sheelio to comment on how we should handle sheets with the old header name.

**Comment by sravfeyn** [comment]
Sheel says this behaviour is OK http://manage.dimagi.com/default.asp?226126#BugEvent.1141542

**Pull request merged**